src/rtp/rtp.c

```diff
@@ -1593,106 +1586,106 @@ static int rtp_recv_data(struct rtp *session, uint32_t curr_rtp_ts)
 static void rtp_process_data(struct rtp *session, uint32_t curr_rtp_ts,
                uint8_t *buffer, rtp_packet *packet, int buflen)
 {
-        /* This routine preprocesses an incoming RTP packet, deciding whether to process it. */
-        uint8_t *buffer_vlen = NULL;
-        int vlen = 12;          /* vlen = 12 | 16 | 20 */
-        source *s;
+        if (buflen <= 0)
+	        return;
```

Comment:
Branch swap of the original master-if. Most of this change is just whitespace change, as the whole branch code was unindented by one level.